lib/pages/diets_page.dart

```diff
@@ -11,30 +14,141 @@ import '../utility/rr_buttons.dart';
+  GagDish(name: 'Джем'),
+  GagDish(name: 'что-то супердлинное что-то'),
+  GagDish(name: 'к'),
+  GagDish(name: 'овсянка'),
+];
+
 class DietListPage extends ConsumerWidget {
   const DietListPage({super.key});
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final constants = ref.read(constantsProvider);
     final asyncDiets = ref.watch(dietsProvider);
-
     return CustomScaffold(
-      title: 'Питание',
-      body: RRSurface(
-        child: Padding(
-          padding: EdgeInsets.all(constants.paddingUnit * 2),
-          child: AsyncBuilder(
-              asyncValue: asyncDiets,
-              builder: (diets) {
-                return ListView.builder(
-                    itemCount: diets.length,
-                    itemBuilder: (context, index) =>
-                        DietTile(diet: diets[index]));
-              }),
-        ),
-      ),
-    );
+        title: 'Питание',
+        body: RRSurface(
+            child: Padding(
+                padding: EdgeInsets.all(constants.paddingUnit * 2),
+                child: ListView.builder(
+                    itemCount: gagList.length,
+                    itemBuilder: (context, index) {
+                      return Container(
+                          padding: EdgeInsets.all(constants.paddingUnit / 2),
+                          margin: EdgeInsets.only(
+                              top: constants.paddingUnit / 2, bottom: constants.paddingUnit / 2),
+                          decoration: BoxDecoration(
+                            border: Border.all(
+                                color: Theme.of(context).colorScheme.surface,
+                                width: constants.borderWidth),
+                            borderRadius: BorderRadius.circular(constants.dInnerRadius),
+                          ),
+                          child: ListTile(
+                              contentPadding:
+                                  EdgeInsets.only(left: constants.paddingUnit, right: 0),
+                              title: StyledHeadline(
+                                  text: gagList[index].date,
+                                  textStyle: Theme.of(context).textTheme.headlineSmall!.copyWith(
+                                        fontWeight: FontWeight.bold,
+                                        height: 1,
+                                        leadingDistribution: TextLeadingDistribution.proportional,
+                                      )),
+                              onTap: () {},
+                              trailing: gagList[index].familyRatio.isEmpty &&
+                                      gagList[index].soloRatio.isEmpty
+                                  ? SizedBox(
+                                      height: double.infinity,
+                                      child: TextButton(
+                                          child: const Icon(
+                                            Icons.add_circle_outline,
```

Comment:
```suggestion
                                      child: IconButton(
                                          icon: const Icon(
                                            Icons.add_circle_outline,
```
Кесарю кесарево, иконкам IconButton